react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -334,8 +353,30 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
         }
 
         if (buttons.length > 0) {
+            if (this.props._participantCount === 2) {
+                // This check was added because the issues was present
```

Comment:
Maybe specify what is the issue and also where the `143` is coming from?

Reply:
Created a variable and added a comment for it.

Reply:
My question was more like:
Why exactly 143? Is this the height of the popup or something else?

Reply:
143 is the height value for the whole filmstrip when it's rendered on small screen.